qiime2/sdk/action.py

```diff
@@ -169,23 +191,17 @@ def callable_wrapper(*args, **kwargs):
 
             outputs = self._callable_executor_(self._callable, view_args,
                                                output_types, provenance)
-            # `outputs` matches a Python function's return: either a single
-            # value is returned, or it is a tuple of return values. Treat both
-            # cases uniformly.
```

Comment:
I noticed we didn't really need this anymore since we always return a `Results` object which means `_callable_executor_` can just always return a `tuple` now!